packages/markdown/processor/parse/flavored/code-tabs.js

```diff
@@ -1,4 +1,4 @@
-const RGXP = /^(```([^]+?)(?=```\n\n)```)\n\n/g; // code blocks
+const RGXP = /^(```[^`]+?```\n(?:```)[^]+?```)/g; // code blocks
```

Comment:
Occasionally, authors don't add proper line breaks between various Markdown blocks. Since our code tab regex was dependent on certain newline sub-patterns, this would end up rendering some wonky shit in these (quite specific) instances. In one example that I ran across, a user had written this:

    ```js
    console.log("I'm supposed to be a single code block.");
    ```
    ## I Should Be A H2 Title
    ```php
    echo "I should also be a plain ol' code block."
    ```

I've pasted a screen shot of the expected result, and the actual render for comparison:

<p align=center><img width=88% src=https://user-images.githubusercontent.com/886627/80321909-f798ee00-87d5-11ea-8479-842dae6ecd6e.png></p>

I've cleaned up the pattern so code tabs are now recognized entirely based on concurrent `` ``` `` back tick tags, which fixed this rare issue with missing newlines!

Reply:
Can you add a test for this case?

Reply:
@erunion added a case for this in 4654fe8